src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/mappers/ExceptionRecordMapper.java

```diff
@@ -28,6 +29,12 @@
     private final String documentManagementUrl;
     private final String contextPath;
 
+    // Display Envelope case references for the specified classifications
+    EnumSet<Classification> allowedClassifications = EnumSet.of(
```

Comment:
this becomes classic private static final member :D

Reply:
thanks, updated